CHANGELOG.md

```diff
@@ -19,21 +19,24 @@
 
 - feat: Add trace filter API supporting RPC method `trace_filter` ([filecoin-project/lotus#12123](https://github.com/filecoin-project/lotus/pull/12123)). Configuring `EthTraceFilterMaxResults` sets a limit on how many results are returned in any individual `trace_filter` RPC API call.
 
-# v1.28.0 / 2024-07-23
-This is the MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
+# v1.28.1 / 2024-07-24
 
-**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.** 
+This is the MANDATORY Lotus v1.28.1 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇. v1.28.1 is also the minimal version that supports nv23.
+**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.**
```

Comment:
@jennijuju do we want to push this out by a week to give more time for SPs to drop the old ProveCommit flow?

Reply:
bit late for that I think, we've pushed people to run v1.28.1, any change is going to require a 1.28.2 and a lot of coercive effort to get people to upgrade before the 8th to prevent forky fork fork